src/lib/coda_lib/coda_lib.ml

```diff
@@ -295,7 +295,7 @@ module type Proposer_intf = sig
     -> time_controller:time_controller
     -> keypair:keypair
     -> consensus_local_state:consensus_local_state
-    -> transition_frontier:transition_frontier
+    -> frontier_ref:transition_frontier Mvar.Read_only.t
```

Comment:
would `_mvar` be a better name than `_ref`; the latter suggests an OCaml ref type